mapproxy/cache/tile.py

```diff
@@ -318,6 +326,12 @@ def is_cached(self, tile):
         """
         return self.tile_mgr.is_cached(tile)
 
+    def is_stale(self, tile):
+        """
+        Return True if the tile is cached.
```

Comment:
Should better read `Return True if tile exists _and_ is expired.` as the original method